api/config/dex/config_request.proto

```diff
@@ -105,6 +105,14 @@ message ConfigRequest {
 			google.protobuf.StringValue base_group_search_dn = 6;
 			google.protobuf.StringValue user_query_filter = 7;
 			google.protobuf.StringValue group_query_filter = 8;
+			google.protobuf.StringValue username_attr = 9;
```

Comment:
Do you have a reference that specifies all these fields? Might be nice to include a link in this file.

Reply:
They eventually map to LDAP config settings in Dex but they aren't 1-1.